src/lib.rs

```diff
@@ -570,7 +570,12 @@ impl<'a> Iterator for Iter<'a> {
     fn next<'b>(&'b mut self) -> Option<&'a error::Error> {
         match self.0.take() {
             Some(e) => {
-                self.0 = e.cause();
+                self.0 = match () {
```

Comment:
Same as above
```rust
                self.0 = {
                    #[cfg(not(has_error_source))]
                    {
                        e.cause()
                    }
                    #[cfg(has_error_source)]
                    {
                        e.source()
                    }
                };
```

Reply:
This review comment still hasn't been resolved.

Reply:
same as above

Reply:
@AndyGauge do you have write access to this repo?

Reply:
@tworec I asked @kodraus on the libs team for write access for this PR.  I'm hoping this ping catches his attention.

Reply:
Ah thanks for the ping! @AndyGauge I’ve added you as a collaborator to this repo.